### DISCUSSION TITLE
Modify PR Merge Commit Based on Presence of Label

### DISCUSSION THREAD

**Discussion by tymokvo** [community/community#26056] [Actions]
<p>Hello,</p>
<p>Just wanted to check to see if this is possible: can I set an option to change the message of a PR’s merge commit based on the presence of a label on that PR? E.g. a label of <code>hotfix</code> results in prepending <code>Hotfix:</code> to the resulting commit message.</p>
<p>Thanks!</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/tymokvo">@tymokvo</a> thanks for stopping by the community forum!</p>
<p>I do believe this is possible, and I think you could benefit from checking out the <a href="https://github.community/c/github-account-questions/42">Actions category</a>.</p>
<p>You may also be able to find a pre-defined Action in the <a href="https://github.com/marketplace?type=actions">Marketplace</a> that would suit your needs.</p>
